app/Http/Transformers/ReportbackItemTransformer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\ReportbackItem;
+use League\Fractal\TransformerAbstract;
+
+class ReportbackItemTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models Reportback $reportback
```

Comment:
This docblock should be updated to `\Rogue\Models\ReportbackItem $reportbackItem`.